pkg/init/Dockerfile

```diff
@@ -13,3 +19,4 @@ COPY --from=mirror /out/ /
 COPY init /
 COPY etc etc/
 COPY bin bin/
+COPY --from=build usermode-helper /sbin/
```

Comment:
I would move that higher to have the `COPY` lines from other stages grouped together.